app/google-services.json

```diff
@@ -0,0 +1,39 @@
+{
+  "project_info": {
+    "project_number": "473213372413",
+    "project_id": "polysmee-e6a86",
+    "storage_bucket": "polysmee-e6a86.appspot.com"
+  },
+  "client": [
+    {
+      "client_info": {
+        "mobilesdk_app_id": "1:473213372413:android:92cf9ee224c66e1b933b94",
+        "android_client_info": {
+          "package_name": "io.github.polysmee"
+        }
+      },
+      "oauth_client": [
+        {
+          "client_id": "473213372413-mltrcpeqlhq3ach5cpjf5epa72mgjkru.apps.googleusercontent.com",
+          "client_type": 3
+        }
+      ],
+      "api_key": [
+        {
+          "current_key": "AIzaSyAqcCJHRFKG-xxYpgaSPTiMmuH8e1M5JNw"
+        }
+      ],
+      "services": {
+        "appinvite_service": {
+          "other_platform_oauth_client": [
+            {
+              "client_id": "473213372413-mltrcpeqlhq3ach5cpjf5epa72mgjkru.apps.googleusercontent.com",
+              "client_type": 3
```

Comment:
IDs directly in plaintext ?

Reply:
yes those are shared across users, that's okay